content/en/monitors/faq/why-did-my-monitor-settings-change-not-take-effect.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Why Did My Monitor Settings Change Not Take Effect?
+kind: faq
+
+---
+
+We will keep monitor groups available in the UI for 24 hours.  If you do not have "No Data" alert settings enabled and your group for a metric monitor stops reporting data, it will persist on the monitor status page until it ages out, though that group will stop being evaluated after a short absence (specific timing depends on your settings).
```

Comment:
```suggestion
Datadog keeps monitor groups available in the UI for 24 hours.  If you do not have *No Data* alert settings enabled and your group for a metric monitor stops reporting data, it will persist on the monitor status page until it ages out, though that group will stop being evaluated after a short absence (specific timing depends on your settings).
```